Dockerfile.gatekeeper

```diff
@@ -8,8 +8,7 @@ ARG BUILDKIT_SBOM_SCAN_STAGE=true
 ENV GO111MODULE=on \
     GOOS=linux \
     GOPATH=/go/ \
-    GOARCH=amd64 \
-    CGO_ENABLED=0
```

Comment:
Is removing the variable equal to CGO_ENABLED=1 in this case?

Reply:
yes
```
$ go version -m manager | grep CGO
	build	CGO_ENABLED=1